docs/features/techdocs/getting-started.md

```diff
@@ -90,10 +90,16 @@ techdocs:
 ```
 
 Setting `generators.techdocs` to `local` means you will have to make sure your
-environment is compatible with techdocs. You will have to install the
-`mkdocs-techdocs-container` and 'mkdocs' package from pip, as well as graphviz
-and plantuml from your package manager. This has only been tested with python
-3.7 and python 3.8.
+environment is compatible with techdocs.
+
+You will have to install the `mkdocs-techdocs-core` and `mkdocs` package from
```

Comment:
What I experienced was that the mkdocs build will error out with incorrect Markdown version unless Markdown==3.2.2 explicitly installed, since the mkdocs package at this time will cause Markdown version 3.3.3 or higher to be installed instead

Reply:
Hmm, I see. `mkdocs-techdocs-core` installs the version `Markdown==3.2.2` but `mkdocs` overwriters `Markdown`'s version to a latest one. This has to do with the order in which we install using pip.

Installing `mkdocs-techdocs-core` after `mkdocs` will solve this problem I think. Will update the docs and add explicit information about `Markdown` version. Thanks again @adrianke77, much appreciated!